test/bdd/fixtures/vct/mysql-config/mysql_config.sql

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright Google LLC, SecureKey Technologies Inc. All Rights Reserved.
```

Comment:
Should adjust this line to link to the upstream project's copyright / license information. 

The next lines are good - URL of original file and SPDX license summary.